.gitignore

```diff
@@ -1,12 +1,26 @@
-*~
+# OS X
 .DS_Store
 
+# Others
+*~
 *.log
-/node_modules
-fixtures/**/node_modules
 
+# VSCode
 .vscode/
+tsconfig.json
 jsconfig.json
+
+# NodeJS
+node_modules/
+npm-debug.log
+yarn-debug.log
+
+# Webpack
+.happypack
```

Comment:
We don't use happypack anymore. If you have the dir, you better remove it (and this entry too) 😄